### PR TITLE
Changing Package Name to Assignment3

### PR DESCRIPTION
**Pull request by michael9j2lee**
Modified application.properties with Rabbit amqp server credentials on local machine.

Modified weather_show to show new data.

Added new request_reservation.html for the form sent success event.

Modified pom.xml with dependencies.

### CODE REVIEW

**Comment by michael9j2lee** [comment]
Please review my changes :)

File: src/test/java/cst438_assignment2/domain/CityServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		//check cityinfo
+		CityInfo expectedResult = new CityInfo(
+				new City( 0 ,"TestCity","TestDistrict", 0,
+						new Country( "TestCountryCode","EX")),
+				new TempAndTime(0,1234,4321));
+		
+		assertThat(expectedResult).isEqualTo( cityService.getCityInfo( "TestCity")) ;
+		
+		verify(cityService, times(1)).getCityInfo(ArgumentMatchers.any(String.class));
+		//verify(weatherService, times(1)).getTempAndTime(ArgumentMatchers.any(String.class));
```
Comment (andyalam) [inline src/test/java/cst438_assignment2/domain/CityServiceTest.java]: Let's delete this commented out code as cleanup?

File: src/main/resources/templates/weather_show.html [hunk 2]
```diff
@@ -2,14 +2,58 @@
...
+			<td th:text="${cityInfo.temp} + F"></td>
+		</tr>
+		<tr>
+			<td>Local Time</td>
+			<td th:text="${cityInfo.time}"></td>
+		</tr>
+	</table>
+	<br></br>
+	<form method="POST" th:action="@{/cities/reservation}">
+		<input type="hidden" id="city" name="city" th:value="${cityInfo.name}"/>
```
Comment (andyalam) [inline src/main/resources/templates/weather_show.html:45]: Neat to see you used a hidden field, I used this as well. It felt strange doing it this way but following the MVC pattern I think it makes the most sense.

File: src/main/java/cst438_assignment3/weather/WeatherService.java [hunk 3]
```diff
@@ -22,25 +24,28 @@
 	
 	
 	// retrieve url and apikey for weather service from application.properties file
-	public WeatherService(
+	public WeatherService(    //1
```
Comment (andyalam) [inline src/main/java/cst438_assignment3/weather/WeatherService.java]: Remove the //1 comment!

**Review by andyalam — APPROVED** [review]
Overall looks good. I left a few comments on cleanup to do and observations. There wasn't much new code added this week so I didn't have much to comment on outside of this!

**Pull request merged**